### DISCUSSION TITLE
Is it useful to create more "spider is missing for brand" issues?

### DISCUSSION THREAD

**Discussion by matkoniecz** [alltheplaces/alltheplaces#9350] [General]
I could create many more for Poland but I assumed that bottleneck is at actually writing spiders, and adding more issues will not help with this. The same for various data issues, I created just small sample of issues for each problem and planned to not report more until reported ones are fixed.

But I see @CloCkWeRX adding many "spider missing" issues so maybe there is actually benefit and I also should create some of this not yet reported ones?
Or do we have enough of them and it would make sense to report more later? Or is there some filtering in created issues that I am not aware of?
If reporting more makes sense - does it apply also "spider XYZ has broken data for field ABC"?

**Comment by CloCkWeRX** [comment]
Ah yeah, so I am a bit spammy with the issue requests.

In general there's a feature request process; and I am... applying "human scale" automation to it (#6328 and #6916).

My goal is to try to increase coverage of the NSI as much as is feasible, or failing that, a discussion area for the collected knowledge of why scraping XYZ corporation's bot hostile website isn't ideal.

Luckily, this is a finite problem, just a very large finite problem.. however a handful of determined hackers probably isn't going to nail it to a significant definition of done in a reasonable timeframe.

I think I'd be personally happy to just close off anything that is:
- less than 10 locations
- a hard work spider

Thus #9348.

**Reply by matkoniecz** [reply]
> Luckily, this is a finite problem, just a very large finite problem.. 

I wonder how fast NSI adds new items and how fast NSI entries are processed here (including rejections under "not worth doing it"?)

**Reply by matkoniecz** [reply]
> Ah yeah, so I am a bit spammy with the issue requests.

oh, that is not a big problem. Just that nonspider issues kind of drowned under them :) 

Though I have good idea what I want to do, and I am more deciding of what I will have time to implement and less looking for more ideas anyway.

**Reply by CloCkWeRX** [reply]
There does seem to be a reasonable feed of new PRs based on NSI new suggestions, others seem to keep an eye on this.

As for accidentally drowning things, yeah, I don't love that.
I know I have made an effort to go through the backlog by "last updated"; and that has had a small amount of value.
I figure the it's-hard-to-scrape-no-one-cares issues will gradually float down the list.

Labelling bots to help tag quality issues/bugs I feel like are the most valuable/most benefit thing we can do - tweaking an existing scraper to be more correct seems very high value.
Just a matter then of making it trivial to filter to those views (link in readmes, or stickied posts)

**Comment by CloCkWeRX** [comment]
(in terms of new spiders, check out the discussion area where I have scraped brand/shop/* from NSI and done some precursor work. I think absolutely add stuff for your local mapping area or that will ultimately benefit you/your local network if mapped)
